lib/name-tamer/version.rb

```diff
@@ -1,3 +1,3 @@
 class NameTamer
-  VERSION = '0.2.3'
+  VERSION = '0.2.9'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.